src/core/utils.ts

```diff
@@ -0,0 +1,110 @@
+import base32 from 'hi-base32'
+import {
+  ALGORAND_ADDRESS_BYTE_LENGTH,
+  ALGORAND_ADDRESS_LENGTH,
+  ALGORAND_CHECKSUM_BYTE_LENGTH,
+  CHECKSUM_ADDRESS_ERROR_MSG,
+  MALFORMED_ADDRESS_ERROR_MSG,
+} from './constants'
+import * as nacl from './nacl/naclWrappers'
+
+/**
+ * Check whether the environment is ReactNative
+ * @returns True if ReactNative, false otherwise
+ */
+export function isReactNative() {
```

Comment:
If we remove the tweetnacl dependency then we no longer need this which is nice.